### DISCUSSION TITLE
Towards "Publication Quality" plots?

### DISCUSSION THREAD

**Discussion by lucas-wilkins** [SasView/sasview#2379] [Ideas]
I don't think it will ever be possible to give everyone the degree of control over plots they need for publication quality plots via the gui. There's just too many things that people would like to do.

However, if we make the plots easily accessible through the ipython prompt / scripting, people can modify plots that way and do what they like with them.

Just a thought.

**Comment by butlerpd** [comment]
This is indeed always the tension: A strong desire for the ability to do publication quality plots on the one hand and the level of resources available coupled with the number of higher priority items (should we fix this bug or add this cool new feature important to some reasonable subset of users or make prettier plots 😃) on the other. Particularly since "those who bring the resources" get to choose what is most important to them.

That said most of what SasView plans is aspirational as we often point out. That is why the roadmap always says: "subject to the availability of resources we will..."  Given the very wide desire to see better plotting, I think the main way to handle this long term low level desire is to try to make sure that when we make changes to plotting we don't move the needle backward (and even better move the needle slightly forward) and do not make it harder to achieve the dream than it is now?... but otherwise don't worry too much about it until/unless someone wants to take that on as a project?

Regarding people doing scripting, I think most already do the plotting whatever way they are used to in python (not even necessarily using matplotlib) so I'm not sure how much the solution here would be helpful? But maybe it is a chicken and egg thing? And given the tools people would use it? Do we know enough "scripters" to get a sense?
